cgi-bin/DW/Controller/Media.pm

```diff
@@ -39,7 +39,17 @@ sub media_manage_handler {
 
     # load all of a user's media.  this is inefficient and won't be like this forever,
     # but it's simple for now...
-    $rv->{media} = [ DW::Media->get_active_for_user( $rv->{remote} ) ];
+    my @media = DW::Media->get_active_for_user( $rv->{remote}, width => 200, height => 200 );
+    my @valid_sizes;
+    map { push @valid_sizes, ( $_, $_ ) } keys %VALID_SIZES;
```

Comment:
Instead of doing this extra work here, why not just change the initial VALID_SIZES assignment from `$_ => 1` to `$_ => $_`?  None of the existing checks care what the value of the hash is, just whether the key exists.